src/dataexplorer/dataexplorer-application/src/components/PatientInfo/DisplayPatientInfo.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { IPatient } from 'types/IPatient';
+import { Row, Col, Card, Table } from 'antd';
+import './PatientInfo.style.scss';
+
+const DisplayPatientInfo = (props: {
+    patient: IPatient;
+    handleClick: any;
+    dataSource: fhir.ResourceBase[];
+}) => {
+    const name =
+        props.patient.name[0].given[0] + ' ' + props.patient.name[0].family;
+    const columns = [
+        {
+            title: 'Skjemanavn',
+            dataIndex: 'schemaName',
+            key: 'skjemanavn',
+        },
+        {
+            title: 'Innsendt',
+            dataIndex: 'submitted',
+            key: 'submitted',
+        },
+        {
+            title: 'ID',
+            dataIndex: 'id',
+            key: 'id',
+        },
+    ];
+
+    return (
+        <>
+            <Row gutter={[1, 40]} justify="center">
+                <Col span={12}>
+                    <Card
+                        className="patient-card"
+                        type="inner"
+                        hoverable
+                        key={props.patient.id}
+                        title={name}
+                    >
+                        <div className="info-container">
+                            <div className="info-left">
+                                <h4>Pnr:</h4>
+                                <span>{props.patient.id}</span>
+                                <h4>Kjønn:</h4>
+                                <span>{props.patient.gender}</span>
+                                <h4>Fødselsdato:</h4>
+                                <span>{props.patient.birthDate}</span>
+                            </div>
+
+                            <div className="info-right">
+                                <h4>Addresse: </h4>
+                                <span>
+                                    {props.patient?.address?.[0]?.line?.[0]}
+                                </span>
+                                <h4>Telefon: </h4>
+                                <span>
+                                    {props.patient?.telecom?.[0]?.value}
+                                </span>
+                                <h4>E-post: </h4>
+                                <span>eksempel@epost.no</span>
```

Comment:
Is this supposed to be in this PR?

Reply:
This was fixed in a previous PR

Reply:
I see i highlighted more lines than intended. I was wondering if line 62 is supposed to be here.